.travis.yml

```diff
@@ -3,6 +3,12 @@ sudo: false
 
 matrix:
   include:
+    - os: linux
+      arch: arm64
+      jdk: openjdk8
```

Comment:
I don't see the value on running in openjdk8 any more. It is end-of-life, and Java 11 can compile with Java 8 strict compliance already.